coalib/bearlib/aspects/Smell.py

```diff
@@ -101,20 +92,80 @@ def func(a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, t, z):
         parameters (maybe by creating more functions or using libraries).
         """
 
+
+@MethodSmell.subaspect
+class MethodLength:
+    """
+    `MethodLength` refers to the number of lines of code in a function or
+    method definition. Depending on the value of `method_length_count_comment`,
+    comments are considered.
+    """
+    class docs:
+        example = """
+        def is_positive(num):
+            if num >= 0:
+                return True
+            else:
+                return False
+        # Here is compact implementation of this method
+
+        def is_positive(num): return True if num >= 0 else False
+        """
+        example_language = 'python'
+        importance_reason = """
+        The longer a method or function is, the harder it becomes to
+        understand and maintain it. Also long methods easily introduce
+        unwanted duplicated code, and bugs to our codebase.
+        """
+        fix_suggestions = """
+        Refactoring methods into smaller more generic methods, making code more
+        compact by using inline and language specific syntax tricks,
+        implementing methods or functions to avoid redundant operation(s),
+        making use of methods provided in libraries rather than reimplementing
+        them.
+        """
     max_method_length = Taste[int](
         'Represents the max number of lines for a method or a function\'s'
         'definition.',
-        (40,), default=40)
+        (5, 40), default=40)
```

Comment:
Here comes better values:
* 30:[the rule of 30 ](https://dzone.com/articles/rule-30-%E2%80%93-when-method-class-or)
* 10: from the RuboCopBear
*60 and 150: [from checkstly](http://checkstyle.sourceforge.net/config_sizes.html)